components/automate-ui/src/app/page-components/job-schedule-form/job-schedule-form.component.ts

```diff
@@ -26,4 +26,8 @@ export class JobScheduleFormComponent {
 
   public minutes = Array(60).fill(1).map((_, i) => i);
 
+  public subControls(name: string): { [key: string]: AbstractControl } {
+    return (this.form.get(name) as FormGroup).controls;
```

Comment:
Needed to do a type cast to correctly access the `controls` property, which is simpler in the code-behind like this.